ISICbalance.xcodeproj/project.pbxproj

```diff
@@ -415,6 +413,24 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		14089B4D223FA11900095BC2 /* SwiftLint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = SwiftLint;
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n\"${PODS_ROOT}/SwiftLint/swiftlint\"\n";
```

Comment:
Swiftlint umí i autocorrect, kterým ti opraví základní chyby (moc mezer, moc enterů, naformátuje některé části kódu správně, ...). Ideálně to nasaď tak, aby se zavolal autocorrect, potom samotný lint a pak se zbuildily všechny zdrojové soubory. (Hint: pouze změň pozici té build fáze trochu výš, ať je to před kompilací)

Reply:
Bohužel se mi ten autocorrect nedaří nasadit. Pokud volám swift lint autoccorect přes terminál (./Pods/SwiftLint/swiftlint autocorrect), tak mi to projde bez problému. Pokud ale volám autocorrect z runscript phase ("${PODS_ROOT}/SwiftLint/swiftlint autocorrect"), tak dostávám error 

> /Users/babacros/Documents/Swift/MI-IOS/ISICkonto/Pods/SwiftLint/swiftlint autocorrect: No such file or directory
Command PhaseScriptExecution failed with a nonzero exit code

Už se to snažím fixnout hodinu a nemlžu najít řešení.

Reply:
neobaluj ty řádky úvozovkami a začne to fungovat 😊

Reply:
Už to běží. 🤦‍♂️ 🤦‍♂️  Děkuji :)